internal/commands/document.go

```diff
@@ -132,11 +133,17 @@ func getDocumentation(path string, outputDirectory string) (map[rego.Severity][]
 		anchor := strings.ToLower(strings.ReplaceAll(documentTitle, " ", "-"))
 		anchor = strings.ReplaceAll(anchor, ":", "")
 
+		matchers := policy.Matchers().String()
```

Comment:
Future thought: The idea with matchers is that we'll grow into other types. In this case, where we want to know if any matchers exist--having to go through each type and checking the length of the slice would be annoying. That said, I wonder if a HasMatchers would be beneficial.

Reply:
Can you elaborate what you mean by other types? I'm not really following this.

Reply:
Yes! So we only support the Kind matcher right now, but GK has other types of matchers. That's why it's currently laid out how it is with the Matcher struct, then a KindMatcher--to allow for other matchers https://github.com/open-policy-agent/gatekeeper#constraints

Reply:
Thanks for clarifying! I agree that in the future we would want to rework this a bit (and not just use `Matchers.String()`) but for now this lets us handle the docs gen for when no KindMatchers are specified without expanding the scope of this PR.

Reply:
Yep exactly, just something to keep in mind as we move forward.